cfpb/cfgov-refresh
# Update hero and fix typo on Financial Well-Being Scale app

A couple little things to make the testing prototype slightly more polished.

## Additions

- Proper hero illustration

## Changes

- Typo in page title fixed

## Testing

1. Pull branch
2. Run server
3. Visit http://localhost:8000/consumer-tools/financial-well-being/
4. See new hero illustration and grammatically correct `<title>`

## Checklist
